draft-ietf-anima-autonomic-control-plane/draft-ietf-anima-autonomic-control-plane.xml

```diff
@@ -1198,31 +1198,44 @@ will be distributed with the flooded objective.  An example of the message is in
 
                                 <t>The following sub-sections define the security association protocols that are considered to be important and feasible to specify in this document.</t>
 
-                        <section anchor="IPsec-group" title="ACP via IKEv2">
+                        <section anchor="IPsec-group" title="ACP via IPsec">
 
-                        <t>An ACP node announces its ability to support IKEv2 as the ACP secure channel protocol in GRASP as "IKEv2".</t>
+<!--kaduk I did not propose to change the GRASP method name here, since I do not know how disruptive that would be.  But the ACP data channel is carried over IPsec, not IKEv2, so from that POV changing the method name would be recommended.-->
+                        <t>An ACP node announces its ability to support IPsec, negotiated via IKEv2, as the ACP secure channel protocol in GRASP as the "IKEv2" method for the "AN_ACP" objective.</t>
 
```

Comment:
The negotiation will occur over IKEv2.  IKEv2 could negotiate something other than ESP. It *could* negotiate MACsec, or many other things.  The important point is that things start on port 500, the rest of up to the negotiation.

Reply:
Supporting what Mcr said: What we have is more precise than saying IPsec as you propose, so i suggest not to adopt this push, unless you have additional supportive arguments for your proposed change.

Also background: Very early on in the design i introduced an option for how we could in the future automatically select between IPsec and non-IPsec (such as MacSec). This was through several review feedback downgraded to appendix A.6 because the argument was that in the face of our Internet security architecture it would be a lot better if we could/would extend IKEv2 to select other data channel options than only IPsec, and IKEv2 was specifically designed to support such extensions. That was when we choose to be more precise and explicit about IKEv2 being the next-protocol after GRASP discovery and then whatever IKEv2 decides is subject to current draft requriements or future extensions.

Reply:
Grmbl... Given how IPsec is a name for the whole enchilada, i fear that if IKEv2 was extended to support negotiation of MacSEC instead of just AH or ESP, then MacSEC would also  be considered IPsec. So let me see how can be most correct about this unfortunate wording of IPsec (as a name for an extensible enchilada).